plugins/home/src/homePageComponents/FeaturedDocs/FeaturedDocs.tsx

```diff
@@ -0,0 +1,173 @@
+  InfoCard,
+  Progress,
+  ErrorPanel,
+} from '@backstage/core-components';
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+import { useApi } from '@backstage/core-plugin-api';
+import { EntityFilterQuery } from '@backstage/catalog-client';
+import { ClassNameMap } from '@material-ui/styles';
+
+import { makeStyles, Theme, Typography } from '@material-ui/core';
+
+/**
+ * Props customizing the <FeaturedDocs/> component.
+ *
+ * @public
+ */
+export type FeaturedDocsProps = {
+  /** The entity filter used to display only the intended item/s */
+  filter: EntityFilterQuery;
+  /** An optional color which can be customized through themes */
+  color?: 'inherit' | 'primary' | 'secondary';
+  /** An optional ClassNameMap created with makeStyles */
+  customStyles?: ClassNameMap<string>;
+  /** An optional ReactNode for empty states */
+  emptyState?: React.ReactNode;
+  /** An optional path to set for entity entry  */
+  path?: string;
+  /** An optional limit to set for link destination  */
+  responseLimit?: number;
+  /** An optional string to customize sublink text */
+  subLinkText?: string;
+  /** An optional string or ReactNode to customize the card title */
+  title?: React.ReactNode | string;
+};
+
+const useStyles = makeStyles<Theme>(
+  theme => ({
+    docDescription: {
+      fontSize: theme.typography.body1.fontSize,
+      fontWeight: theme.typography.body1.fontWeight,
+      marginBottom: theme.spacing(2),
+      marginTop: theme.spacing(2),
+    },
+    docSubLink: {
+      fontSize: theme.typography.subtitle1.fontSize,
+      fontWeight: theme.typography.subtitle1.fontWeight,
+      lineHeight: theme.typography.subtitle1.lineHeight,
+    },
+    docsTitleLink: {
+      fontSize: theme.typography.h6.fontSize,
+      fontWeight: theme.typography.h6.fontWeight,
+      lineHeight: theme.typography.h6.lineHeight,
+    },
+  }),
+  { name: 'BackstageFeaturedDocs' },
+);
+
+/**
+ * A component to display specific Featured Docs.
+ *
+ * @public
+ */
+export const FeaturedDocs = (props: FeaturedDocsProps) => {
+  const {
+    color,
+    customStyles,
+    emptyState,
+    filter,
+    path,
+    responseLimit,
+    subLinkText,
+    title,
+  } = props;
+  const linkText = subLinkText || 'LEARN MORE';
+  const defaultStyles = useStyles();
+  const styles = customStyles || defaultStyles;
+  const catalogApi: CatalogApi = useApi(catalogApiRef);
+  const {
+    value: entities,
+    loading,
+    error,
+  } = useAsync(async () => {
+    const response = await catalogApi.getEntities({
+      filter: filter,
+      limit: responseLimit || 10,
+    });
+    return response.items;
+  });
+
+  if (loading) {
+    return <Progress />;
+  }
+  if (error) {
+    return <ErrorPanel error={error} />;
+  }
+
+  return (
+    <InfoCard variant="gridItem" title={title || 'Featured Docs'}>
+      {entities?.length
+        ? entities.map(d => (
+            <div key={d.metadata.name} data-testid="docs-card-content">
```

Comment:
nitpick: the key should probably be the full entity ref `stringifyEntityRef(d)` since names can be duplicated

Reply:
I thought `metatdata.name` must be unique across kinds. Is this documentation outdated? https://backstage.io/docs/features/software-catalog/descriptor-format#name-required

Reply:
"per kind, within a given namespace"

So the name + namespace + kind triplet is what needs to be unique. In other words, there may be two users named `freben`, in two different namespaces.

Reply:
Got it thank you for clarifying, updated 👍